code/game/objects/items/robot/items/food.dm

```diff
@@ -60,7 +60,12 @@
 		if(DISPENSE_LOLLIPOP_MODE)
 			food_item = new /obj/item/food/lollipop/cyborg(turf_to_dispense_to)
 		if(DISPENSE_ICECREAM_MODE)
-			food_item = new /obj/item/food/icecream(turf_to_dispense_to, list(ICE_CREAM_VANILLA))
+			food_item = new /obj/item/food/icecream(
+				/* loc = */ turf_to_dispense_to,
+				/* starting_reagent_purity = */ null,
+				/* no_base_reagents = */ FALSE,
+				/* prefill_flavours = */ list(ICE_CREAM_VANILLA),
```

Comment:
get rid of these comment markers, just used actual named arguments

Reply:
These named arguments we _already have_ in the codebase are runtiming (so I need to make them not named in this PR). I swear to you this shit doesn't work.
![image](https://github.com/tgstation/tgstation/assets/7483112/9a176bd1-3901-4fdc-943a-9502429605bd)

Reply:
discussed w/ spaceman